### PR TITLE
adding temp/humidity sensors for raritan px2

### PR DESCRIPTION
**Pull request by sjtarik**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
@sjtarik Hey, thanks for the change.  It looks like this could be moved to yaml (which we prefer).  Any reason you didn't use that?

**Comment by sjtarik** [comment]
@murrant Raritan in this new version treat all sensors as generic external sensors and I had to programmatically identify whether a sensor is temperature or humidity instead of using fixed OIDs.

**Comment by murrant** [comment]
This is actually possible via yaml too, using skip_values to check externalSensorType.
https://docs.librenms.org/Developing/os/Health-Information/

Also, it is smart enough that if you specify the same table for multiple sensors it only walks it once.  And if that isn't enough, you can specify pre-cache in yaml too.

**Comment by sjtarik** [comment]
interesting. to decide if fahrenheit --> celcius is needed, how do you check an OID and programatically set the conversion function?

**Comment by murrant** [comment]
Ah, you got me.  You can define a function to run, but you can't set it based on another value.

**Comment by sjtarik** [comment]
@murrant I have two more device vendors to go after that I can take a look at implementing that feature into yaml workflow?

File: includes/discovery/sensors/humidity/raritan-pdu.inc.php [hunk 1]
```diff
@@ -41,10 +40,8 @@
         $measure_data = $pre_cache['raritan_extSensorMeasure'][$index];
         $current      = ($measure_data['measurementsExternalSensorValue'] / $divisor);
         $sensor_available = $measure_data['measurementsExternalSensorIsAvailable'];
-        d_echo("Raritan pdu stuff ".$current." ".$sensor_available." \n");
-        d_echo("Raritan pdu stuff ".$oid."\n");
         $user_func = null;
-        if (is_numeric($current) && $current >= 0 && sensor_available == true) {
+        if (is_numeric($current) && $current >= 0 && $sensor_available == true) {
```
[the remark this answers is not in the document]
Comment (sjtarik) [reply includes/discovery/sensors/humidity/raritan-pdu.inc.php]: Should I use "sensor_available === "true" instead?

**Pull request merged**